src/lib/modules/charts/components/model-modal/model-modal.component.ts

```diff
@@ -40,6 +40,7 @@ export class ModelModalComponent implements OnInit {
 
     ngOnInit() {
         this.modalOptions = Object.assign({}, this.DEFAULT_MODAL_OPTIONS, this.config);
+        this.climateModels = [];  // initialize with empty list while modal loads
```

Comment:
It should be possible to initialize the `@Input` declaration with the empty array, rather than a separate line in OnInit to do so, e.g.:
```
@Input() models: ClimateModel[] = [];
```